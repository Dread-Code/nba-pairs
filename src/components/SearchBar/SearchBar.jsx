import React from 'react'
import { Search, Grid, Header, Icon } from 'semantic-ui-react'
import './SearchBar.scss'

const SearchBar = () => (
  <Header block>
    <Grid doubling>
      <Grid.Column width={4} verticalAlign="middle">
        <Icon name="basketball ball" size="big" color="orange" />
        <Header.Content>NBA PAIRS</Header.Content>
      </Grid.Column>
      <Grid.Column textAlign="center" width={8}>
        <Search size="large" />
      </Grid.Column>
      <Grid.Column width={4} floated="left">
        <a href="https://www.macheight.com/">
          <img
            className="searchbar-match-icon"
            src="//images.squarespace-cdn.com/content/v1/5f7c74b3ebc1ba132269c5a7/1611678115046-8C180XU5WIJ07PA60ADA/mach_8_logo_highlight_full.jpg?format=1500w"
            alt="Mach Eight, LLC"
          />
        </a>
      </Grid.Column>
    </Grid>
  </Header>
)

export default SearchBar
