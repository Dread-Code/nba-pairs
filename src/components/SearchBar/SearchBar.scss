.ui.block.header {
  background-color: black !important;
  .content {
    color: white !important;
    font-size: 2vw;
  }
  img {
    height: 4vh !important;
    border-radius: 3vh;
  }
}
