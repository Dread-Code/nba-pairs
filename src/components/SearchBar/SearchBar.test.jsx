import React from 'react'
import { shallow } from 'enzyme'
import SearchBar from './SearchBar'
import { PlayerContext } from '../../Context/PlayersContext'

describe('SearchBar', () => {
  test('should match to snapshoot', () => {
    const wrapper = shallow(
      <PlayerContext.Provider>
        <SearchBar />
      </PlayerContext.Provider>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
