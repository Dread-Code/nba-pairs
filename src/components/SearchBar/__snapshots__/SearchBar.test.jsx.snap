// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBar should match to snapshoot 1`] = `<SearchBar />`;
